app/src/main/java/io/webthings/app/ui/screens/GwSettingsScreen.kt

```diff
@@ -0,0 +1,22 @@
+package io.webthings.app.ui.screens
+
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.tooling.preview.Preview
+import io.webthings.app.ui.NavRoutes
+import io.webthings.app.utils.MainViewModel
+
+@Composable
+fun GwSettingsScreen(viewModel: MainViewModel){
+    viewModel.setCurrentScreen(NavRoutes.DrawerItems.GwSettingsScreen)
+    /*
+     * TODO
+     */
```

Comment:
i'll use TODO because it automatically show in the android studio todo list, so later i'll jsut go trough this list to fix

when creating my abstract class, i use todo because i have to create the base of things class without completing it already, it will came in a next pr